src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const persons = tbody.querySelectorAll('tr');
+
+thead.addEventListener('click', (e) => {
+  let sortedPersons;
+
+  switch (e.target.textContent) {
+    case 'Name':
+      sortedPersons = [...persons]
+        .sort((a, b) => {
+          return a.children[0].textContent
+            .localeCompare(b.children[0].textContent);
+        });
+
+      sortedPersons.forEach(person => tbody.append(person));
+      break;
+    case 'Position':
+      sortedPersons = [...persons]
+        .sort((a, b) => {
+          return a.children[1].textContent
+            .localeCompare(b.children[1].textContent);
+        });
+
+      sortedPersons.forEach(person => tbody.append(person));
+      break;
+    case 'Age':
+      sortedPersons = [...persons]
+        .sort((a, b) => {
+          return a.children[2].textContent
+            .localeCompare(b.children[2].textContent);
+        });
+
+      sortedPersons.forEach(person => tbody.append(person));
+      break;
+    case 'Salary':
+      sortedPersons = [...persons]
+        .sort((a, b) => {
+          return a.children[3].textContent
+            .localeCompare(b.children[3].textContent);
+        });
+
+      sortedPersons.forEach(person => tbody.append(person));
```

Comment:
You do this logic on every case. You can just do it after the switch

Reply:
+